services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/organisaatio/OrganisaatioController.kt

```diff
@@ -17,8 +18,11 @@ class OrganisaatioController(@Autowired private val service: OrganisaatioService
 
     @GetMapping
     fun getOrganisaatiot(): ResponseEntity<Any> {
+        logger.info {
+            "Getting organisaatiot..."
+        }
         val organisaatiot = service.getOrganisaatiot()
-        logger.info { organisaatiot }
+        logger.info { "Got organisaatiot." }
         return ResponseEntity.status(HttpStatus.OK).body(organisaatiot)
```

Comment:
Good at this point. It at some point might be worth to add e.g. how many organizations we got.. But no need at the moment. Also maybe that would be for the debug level logging?